crates/turborepo-lib/src/turbo_json/mod.rs

```diff
@@ -73,7 +76,61 @@ pub struct RawTurboJson {
     pub(crate) remote_cache: Option<ConfigurationOptions>,
 }
 
-#[derive(Serialize, Deserialize, Default, Debug, PartialEq, Clone)]
+#[derive(Debug, Default, Clone, Serialize)]
+#[serde(transparent)]
+pub struct Spanned<T> {
+    value: T,
+    #[serde(skip)]
+    range: Option<Range<usize>>,
+    #[serde(skip)]
+    path: Option<Arc<str>>,
+    #[serde(skip)]
+    text: Option<Arc<str>>,
```

Comment:
Ehh all of these are kinda similarly grouped, like text, path, and range. Each of them are optional because they get added to the data at different points and it's possible for them to be missing.